package.json

```diff
@@ -37,7 +39,9 @@
     "test": "node node_modules/.bin/jest"
   },
   "jest": {
-    "testMatch": ["<rootDir>/src/**/__tests__/*.js"]
+    "testMatch": [
+      "<rootDir>/src/**/__tests__/*.js"
+    ]
```

Comment:
That was not me, I swear :) that's prettier!

Reply:
ha ha :P